hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/consumer/Message.java

```diff
@@ -53,6 +53,8 @@ public class Message implements FilterableMessage {
 
     private long currentMessageBackoff = -1;
 
+    private boolean isFiltered = false;
```

Comment:
we must ensure that the this value is visible to all threads (see the comment on top of this class)

Reply:
I have checked it and this is always used within a single thread.